atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractGenericStreamStore.java

```diff
@@ -139,7 +139,7 @@ private int getNumberOfBlocksThatFitInMemory() {
     }
 
     @Override
-    public final File loadStreamAsFile(Transaction transaction, ID id) {
+    public File loadStreamAsFile(Transaction transaction, ID id) {
```

Comment:
Correct, good catch. A previous version only overrode this method and copied a bunch of other code to avoid making private methods protected. I switched to the current version (only overriding tryWriteStreamToFile) to change and duplicate as little code as possible.